### PR TITLE
Remove some last traces of Trac

### PR DESCRIPTION
**Pull request by mkoeppe**
### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [ ] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by GiacomoPope** [comment]
Is it worth replacing `:trac:` for `:issue:` in the doctests? Apologies if this has already been done.

**Comment by mkoeppe** [comment]
> Is it worth replacing `:trac:` for `:issue:` in the doctests?

Not in this PR, but sure, why not

File: .gitpod-setup-trac-remote.sh [hunk 1]
```diff
@@ -4,26 +4,5 @@
 set -e
 
 # Setup trac as remote
-## In order to push to trac, generate a new key with `ssh-keygen -f tempkey` and save the private key to gitpod `gp env PRIVATE_SSH_KEY="$(<tempkey)"` (or by following https://www.gitpod.io/docs/environment-variables#using-the-account-settings)
-## then follow https://doc.sagemath.org/html/en/developer/trac.html#linking-your-public-key-to-your-trac-account to register the public key with trac.
-## Afterwards, create a new gitpod workspace.
-git remote remove trac 2> /dev/null || true # might still exists from a previous run/prebuild
-if [[ -n "${PRIVATE_SSH_KEY}" ]]; then
```
Comment (kwankyu) [inline .gitpod-setup-trac-remote.sh:11]: Line 10 may still be useful?

**Comment by kwankyu** [comment]
Otherwise lgtm.

File: .gitpod-setup-trac-remote.sh (hunk 1 above)
Comment (mkoeppe) [reply .gitpod-setup-trac-remote.sh:11]: Might be. I'll restore it.

**Review by kwankyu — APPROVED** [review]
Thanks. LGTM.

**Comment by mkoeppe** [comment]
Thanks!

**Pull request merged**